app/views/idv/_acuant_sdk_selfie_capture_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<% if Figaro.env.acuant_sdk_document_capture_enabled == 'true' %>
```

Comment:
should this just be a method on `FeatureManagement`?

Reply:
this was hoops

Reply:
the erb meta was hoops too

Reply:
did some refs on acuant_sdk_document_capture_enabled looks like it's only used 2x